index.html

```diff
@@ -23,10 +23,14 @@
       subMaxLevel: 4,
       search: 'auto',
       auto2top: true,
+      alias: {
+        '.*?/pytemplate':
+          'https://raw.githubusercontent.com/Flow-Launcher/Flow.Launcher.Plugin.PythonTemplate/master/README.md',
+      },
       plugins: [
         function (hook, vm) {
           hook.beforeEach(function (html) {
-            url = 'https://github.com/Flow-Launcher/Docs/blob/main/' + vm.route.file
+            url = 'https://github.com/Flow-Launcher/docs/blob/main/' + vm.route.file
```

Comment:
Actually, it would get a problem when docs come from different source. 
So would get a 404 error when click **edit this page**.
There need to add some if-else code but not a big trouble.

Reply:
fixed, see f82cbd9